client/src/App.vue

```diff
@@ -25,6 +25,9 @@
         :redirect-on-logout="getNameRequestUrl"
       />
       <router-view />
+      <transition-group name="fade">
+        <v-btn v-for="(letter, i) of word" @click="word = 'yap'" :key="'letter'+i">{{ word }}</v-btn>
```

Comment:
Is this (and the declaration for `word` debugging code?

Reply:
ooops, i was testing a transition group to make sure i was using it correctly. Im going to rebase and force push this out of the PR